client/components/MessageCenter/MessageCenter.tsx

```diff
@@ -0,0 +1,41 @@
+import './MessageCenter.scss'
+import React from 'react'
+import { useTopics } from '@client/store/ui/messageCenter'
+import { MessageTopic } from '@meta/messageCenter'
+
+type TopicProps = {
+  topic: MessageTopic
+}
+
+const Topic: React.FC<TopicProps> = ({ topic }) => {
+  return (
+    <div className="topic">
+      <div className="topic-header">{topic.key}</div>
+      <div className="topic-body">No comments</div>
+      <div className="topic-footer">
+        <textarea />
+        <div>
+          <button className="btn-s btn-secondary" type="submit">
+            Add
+          </button>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+const MessageCenter: React.FC = () => {
+  const topics = useTopics()
+
+  if (!topics) return null
```

Comment:
this won't work if your initial state is an empty array. check if it's empty ?

Reply:
import { Objects } from '@core/utils'